examples/window/always_on_top.rs

```diff
@@ -0,0 +1,29 @@
+//! Shows how to display a window which stays on top of others.
+//!
+//! This feature works as expected depending on the platform. Please check the
```

Comment:
```suggestion
//! This feature only works on some platforms. Please check the
```

Reply:
I used this wording since its the same wording as the transparent window example: https://github.com/bevyengine/bevy/blob/1914a3f288a812667f735ee9e7806cf1560dacc0/examples/window/transparent_window.rs#L3